### DISCUSSION TITLE
Inner Development Goals and Lene Rachel Andersen

### DISCUSSION THREAD

**Discussion by asimong** [life-itself/community#1154] [Research]
Rufus shared a link to [her statement concerning the Nordic Secret](https://www.youtube.com/watch?v=P8C9A-9P-pQ) uploaded 2024-12-20. What are the different perspectives on this, and what questions arise, potentially for research?

**Comment by JonahWilberg** [comment]
To expand a little on the comment I made in the chat, I think there is quite a rich line of research here around differentiating between authoritarian and non-authoritarian ways of setting inner development goals. This could draw on for example

- history of previous authoritarian cultural movements ('Soviet man', the Chinese cultural revolution, Nazism)
- liberalism vs communitarianism debates in analytic political philosophy
- bildung tradition going back to German romanticism, including Andersen's book
- stage theories as descriptive theories individual and cultural development

**Reply by asimong** [reply]
Thanks @JonahWilberg I appreciate your approach here, and I agree. I would introduce a further distinction … I see two distinct ways of being what you term "non-authoritarian". The first would be a purely relativist position with a strong post-modern flavour, essentially saying (along with [Brian, in the Life of Brian](https://www.youtube.com/watch?v=KHbzSif78qQ)) "you've all got to work it out for yourselves" with no one's view better than anyone else's.

The second, which I would personally much prefer, is that we can help each other with our inner development. Agreements around behaviour do really matter, and these agreements are best arrived at through a consensus-oriented, supportive approach, where not only is everyone heard, but they also hear the opinions and perspectives of others, and take those seriously into account.

**Comment by asimong** [comment]
And for ease of reference I'd like to reproduce here my two comments I've left on YouTube...

> I very much welcome this statement of clarification, pointing towards Bildung as "an open-ended and liberating philosophy", which doesn't seem to match the frame in which the UN SDGs are commonly viewed.  I have slightly different issues with the IDGs and indeed the SDGs, which I hope are seen as complementary, building on, not in opposition to, this statement.
>
> First, I do see the IDGs as pointing to areas of potential development that seem complementary to the SDGs. One can argue – many have – that outer "development" tends not to be effective without some kind of internal change — whether you call it development, growth, or some other name. The point that I see is that structures and patterns "out there" tend to be mirrored "inside", and where this is the case, we need some way to change those inner patterns, or else run the real risk that the internalised patterns will just be recreated. Orwell's "Animal Farm" comes to mind. 
>
> As I was not at all aware of the process which led to the SDGs, I don't know what the intention was, but it's easy to see the result as putting forward an agreed set of goals that we all, collectively, need to be working towards. However, on closer examination, we don't all agree even on these. I have heard in particular strong doubts about Goal 8. 
>
> So I see that we could go two ways. We can take on this welcome statement at a explicit level, and recognise that inner development is much more personal , and that there cannot be fixed goals to produce "ideal human products" (which I find absurd!) There can, however, well be stories of personal growth and development that can be shared as inspiration, and such stories can be found in any religion, any wisdom tradition, as well as in contemporary psychology.
>
> On the other hand, we could instead take the SDGs, not as agreed goals, but as causes which different people adopt and dedicate their efforts towards, without the implication that they are complete, exhaustive, or canonical. Another way of saying this would be, take the spirit of this statement, and apply it to the SDGs as well.  Seems to me this would be greatly helped by altering the word "Goals" in SDGs as well as IDGs.
>
> Earlier this year I did some work comparing different proposed sectors of "regeneration". Although there is a remarkable degree of coherence between different people's interpretations, I don't believe we should be trying to find a "perfect" or agreed definition of what they are, but I do very much support the effort to cross map these (along with SDGs and IDGs) as part of an unfolding, living matrix or potential change, growth, development, transition; so that we can each see and recognise the value in the diverse callings that we have in making this world a better place for all people and all life.

AND subsequently

>  @lene-rachel-andersen  thanks for your reply! I agree that one way of looking at this is to distinguish outer and inner, and to accept goals for the outer while rejecting them for the inner. The point I was making is an attempt to go beyond this dichotomy, and propose an alternative way to respond to this. We agree that goals for inner development can only legitimately be taken on personally and voluntarily, because no one has the authority to set such goals for another. I'm suggesting that it would be helpful to apply the same reasoning to the SDGs themselves. It is at least clear that not everyone agrees on all of the SDGs, so how about, instead of accepting the "goals" that some UN process has come up with, take these as inspiring causes that an individual can adopt for themselves. 
>
> I would go on to suggest that it is futile and counterproductive for anyone to tell anyone else that they should be giving their energy to a particular SDG. Thus, the UN's list would stop being normative, and instead itself be an inspiration for everyone to propose, set, adopt, and work towards the SDGs of their choice; the ones they are called to address, and not be limited by the UN's definition.
>
> Let's take the conversation a bit further. If we see inner development as just a matter of personal choice, we would have to accept personal goals, for example,  to develop "dark triad" traits — I would see this as very unwelcome. So with both inner and outer development, I would suggest that we need careful reflection and dialogue around both the desirability and feasibility of any proposed goals. So I'm saying, not the "goals suck", but that goals (in any complex domain) are inextricably linked to assumptions, worldview, paradigms and personal perspectives, and as such they would all benefit from collective attention, questioning, refinement, and in general consideration.

**Comment by asimong** [comment]
Her [session with Vervaeke](https://www.youtube.com/watch?v=BRlqE14Z7YE) from 2022 is also very interesting — all about Bildung.

**Reply by asimong** [reply]
Which again leaves me with a twin sense: on the one hand, this really is all good stuff; and on the other hand, is that where it stops? I can't help feeling there's something vital missing here, some deeper level maybe? Very crudely, it feels to me like Andersen, and Bildung generally, gives us great lessons in how to be self-authoring, to get from "red" and "blue" to "orange", and *maybe* "green", but without saying much, if anything at all, about the more subtle, later steps, however you characterise them...

**Comment by martinprogrammer** [comment]
I've just watched Lene talk about Bildung

https://www.youtube.com/watch?v=KAOTQ2R0qLg

This is my short summary and related questions.

The correctness of one's morals is measured by "being free" or being able to handle "political freedom".

The society goes through a 3 stage process from:

1) Emotional person
in the throes of emotions
Why is this person "an emotional person" to start with?  What kind of person reacts to frustration emotionally rather than rationally?
Is this a permanent state or a phase.  Can one grow out of it - or does the state always has a pull on that person.

"We can grow out of this - we can learn to be a team player and let the norms of society, the expectations overrule our emotions"
What's the role of sense-making in this self-growth?

2) Rational person - the person who make the rules of society his own. (He's not free because he can't feel his emotions?)
Is this a real self-growth or a conditioned state?  What's the process of adopting the rules of the society to such an extent?  
Developing an alter ago?

3) Moral person - freedom, autonomy from others but also be a team player on behalf of society

Why is the emotional guy in a frenzy of bloodlust that won't stop?
Why would  rational person take clues from others and then choose to follow the guidance of the emotional man?

To calm down emotions we need soothing, calming arts.  That's how we become rational citizens.  How?

"Then the rational person listening to others needs some aesthetics that will make us angry and become involved and engaged and we become a moral person."

I think that she's referring to the society and not necessarily groups of people that fall into these 3 groups.
But, is there any self growth on a societal level?  

The rationale for self growth expressed is normativism - both for the emotional and rational person.
The rationale for the rational man to become a moral person is "authenticity" and "self-expression".  But this "authenticity" is "tainted" here by normativity - so is there any freedom?

The self growth that I can associate with is the one driving towards the exploration of absolute potentials and forcing new postive and progressive paradigms through  escaping from the cage of normativity (structure agency).  So how can escaping the rules of the current system (society) be good?  Well, it's because the assumption is that the current paradigm is NOT GOOD.  So, overcoming NOT GOOD is good.

**Reply by asimong** [reply]
thanks Martin! Reading this (and watching a little of that video) brings me the sense that Bildung, as conceived by Schiller et al, stops at the same place Jung reaches in "individuation", or indeed the same place as (Life of) Brian. That seems to me to correspond with Kegan's 4th order... self-authoring. I'm not sure that the move from Postmodern to Metamodern, or Kegan's 4→5 is clear here. Okay "also a team player" … but I see it in two stages, first being able to relativise self and society, and make decisions (but this I see perhaps as a development within modernism) and then, second, actually being able to co-create and co-direct teams, being a "team player" in a bigger sense. Like when you are actually part of a worker coop. 

Anyway, maybe this is just agreeing with you from another perspective :smile: Sure, escaping the "cage of normativity" is good, and is a vital step in growth, but is that really more than individualism, which is already encompassed by modernism and "Brian"? Personally, I keep looking for something beyond individualism, which is more collective — not where the collective binds individuals, but where the collective and the individual are in mutual synthesis (to use the Peckham Experiment phrase)

**Reply by martinprogrammer** [reply]
I'm looking for something beyond individualism but unconstrained by normativism - that something is collectivisation of authentic individuals with capacity to innovate new paradigms in the positive direction.
But self-growth happens first and the next step (which is the continuation) is the need to form "meaningful relationship" and closely collaborate.

Jaspers touches on that in his musings on communication:

"Not merely and exchange of words , nor friendliness and sociability, but only the constant urge towards total **revelation** reaches the path of communication."

and then

"Alone, I sink into gloomy isolation - only in community with other can I be **revealed in the act of mutual discovery**.  My own freedom can only exist if the other is also free.  Instead or self-isolating Being remains mere potentiality or disappears into nothingness."

**Reply by asimong** [reply]
Very nice, yes! I pause a little with "But self-growth happens first". It's my first- and second-hand experience that a lot of vital "self-growth" is effectively stimulated by the environment, and in particular, by an environment of what Jaspers seems to be talking about. It really isn't, to me, a case of "go away, do your self-growth, and come back when you are mature like us" (I'm sure that's not what you're saying, but some other people come across like that) but sometimes more like "join us, recognise what you are doing, partly through what happens, partly through our prompts to awareness, and answer the call to open up and grow"

Thus I see either the individual or the collective leading. Who leads, what is "first" is much less important than the direction of outcome.

**Reply by martinprogrammer** [reply]
I don't think that self-growth can be choreographed and I don't think it happens to everyone.

But I do think that a collective of people presenting a force for a positive change must come from people who have already been formed through self-growth.  The rest will hopefully be "pushed" into self-growth by the enlightened core.

For me it's important to try and acknowledge that the effort should be made in the direction of attracting individuals who have capacity and have gone through individualistic self-growth and then met an "existential frustration" (like I've been experiencing) where the only way out seems to be through re-inventing relationships and finding better protocols for communication and coherence.

This bringing people together is what can create power (in the current paradigm) and force (the creative lever of change)  I think of one’s inability to do what one would like to do in a positive and progressive direction - as a frustration.  That frustration is the lack of capacity/capabilities to “move big things” is what closer connection, filtering protocol, introduction protocol, coherence protocol, trust protocol… would solve.

**Comment by martinprogrammer** [comment]
It's maybe useful stating what we consider to be self growth and  how it happens.

I like to think of it as improving oneself internally (emotions, resiliance, open mindedness...)  and externally (better relationships with people, empathy, relationship with nature...) through a deeper self-awareness and ability to create novel ways of sense-making, problem solving and interacting with the world.

New neural pathways can be opened by drugs, meditation, traumatic experiences, love, spiritually ...

I also think it's useful to point at transcendence as being that guiding star and maybe the ultimate goal.

**Reply by asimong** [reply]
Good starting point. How do we appreciate or value "improvement"? Is "improvement" a purely personal thing, a purely societal norm thing, or where in between, and how would we know? What is a better relationship? My own experience is that a better relationship for one person can be a worse relationship for another.

To me, this has the same tricky hallmarks as the IDGs themselves. That's not a bad thing, just a note that these things may be impossible to define, and what happens when we actually don't agree on what is better?

I suspect we all have a tendency to see "better" either as "more like me" or as "more like I would like to be". But, again, what if I would like to be a fascist dictator? Who is to judge that that's not a legitimate "better"? We might invoke a principle of equality, saying that it is impossible for many people to be fascist dictators, as there is an exclusivity to that idea, but then, which principles can we all accept and agree on as putting boundaries on what can be considered as "better"?

Maybe we are into serious moral philosophy here, and I'm not a specialist, I just ask questions …

p.s. if I want an example of what someone thought as better, with which I strongly disagree, I need look no further than Ayn Rand.

**Reply by martinprogrammer** [reply]
> p.s. if I want an example of what someone thought as better, with which I strongly disagree, I need look no further than Ayn Rand.

I don't think that anyone's philosophy is necessarily something to agree or disagree with.  Especially in case of Ayn Rand (which is almost a comical mish-mash of different ideas).  Those ideas, put together in a particular fashion, are useful for giving you insights, some bits setting the scene for the "arrival to the ceremony" - gently nudging or turning you at a certain angle to allow you to consider some thoughts more authentically.

We'll soon have an opportunity to witness a real life ideological experiment strongly influenced by Rand  and Nozick ...

**Reply by martinprogrammer** [reply]
@asimong 

Regarding judging "improvement"  - making progress doesn't lend itself to being evaluated against a set of quantitative/qualitative parameters.

Initially, it's just a willingness (or capacity) to change and do a perspective switch.
Focusing on introspection and increasing self-awareness would enable you to figure out the goals and identify limitations.
Developing ability to "walk in someone else's shoes" will engage empathy and will change the nature of your relationships.  That doesn't mean that your relationship with everyone would necessarily become better.  Some of the relationships would stop making sense in the new paradigm.

This is the awakening bit - the next part of the self growth is developing skills and overcoming obstacles to achieve one's goals.
Ultimately, you are on the path of self-actualisation and the realisation of the potential.
